src/pynwb/tests/integration/test_ndx_nirs.py

```diff
@@ -74,6 +74,7 @@ def create_fake_nirs_device():
         name="device",
         description="An fNIRS device",
         manufacturer="XYZ",
+        nirs_mode="time-domain",
```

Comment:
Should we also put some of the new attributes here? e.g. 
```suggestion
        nirs_mode="time-domain",
        time_delay=1337
        additional_parameters="this nirs device has a flux capacitor gain over 9000"
```

Reply:
:laughing: :heart: the flux capacitor. Good idea!